x-pack/plugin/gpu/src/internalClusterTest/java/org/elasticsearch/xpack/gpu/GPUIndexIT.java

```diff
@@ -36,6 +37,11 @@
 public class GPUIndexIT extends ESIntegTestCase {
 
     public static class TestGPUPlugin extends GPUPlugin {
+        @Inject
+        public TestGPUPlugin(Settings settings) {
```

Comment:
We should not need the at Inject here either. What if we remove it, and just have a no-args constructor? The body of the code can just create a new Settings and then add use_gpu ?